CONTRIBUTING.md

```diff
@@ -1,34 +1,232 @@
-tox run -e unit          # unit tests
-tox run -e integration   # integration tests
-tox                      # runs 'format', 'lint', 'static', and 'unit' environments
-```
+* [Using the issue tracker](#using-the-issue-tracker)
+* [Issues and Labels](#issues-and-labels)
+* [Bug Reports](#bug-reports)
+* [Enhancement Proposals](#enhancement-proposals)
+* [Pull Requests](#pull-requests)
+* [Discussions](#discussions)
+* [Code Guidelines](#code-guidelines)
+* [License](#license)
 
-## Build the charm
+## Using the issue tracker
 
-Build the charm in this git repository using:
+The issue tracker is the preferred way for tracking [bug reports](#bug-reports), [enhancement proposals](#enhancement-proposals),
+and [submitted pull requests](#pull-requests), but please follow these guidelines for the issue tracker:
 
-```shell
-charmcraft pack
-```
+* Please __do not__ use the issue tracker for personal issues and/or support requests.
+The [Discussions](#discussions) page is a better place to get help for personal support requests.
 
-<!-- You may want to include any contribution/style guidelines in this document>
+* Please __do not__ derail or troll issues. Keep the discussion on track and have respect for the other
+users/contributors of the hpc-libs charm libraries.
+
+* Please __do not__ post comments consisting solely of "+1", ":thumbsup:", or something similar.
+Use [GitHub's "reactions" feature](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
+instead.
+  * The maintainers of the hpc-libs charm libraries reserve the right to delete comments
+  that violate this rule.
+
+* Please __do not__ repost or reopen issues that have been closed. Please either
+submit a new issue or browser through previous issues.
+  * The maintainers of the hpc-libs charm libraries reserve the right to delete issues
+  that violate this rule.
+
+## Issues and Labels
+
+The issue tracker uses a variety of labels to help organize and identify issues.
+Here is a list of some of these labels, and how the maintainers of the repository use them:
+
+* `Type: Bug` - Issues reported in the source code that either produce errors or unexpected behavior.
+
+* `Status: Confirmed` - Issues marked `Type: Bug` that have be confirmed to be reproducible on a separate system.
+
+* `Type: Documentation` - Issues for improving or updating the documentation.
+Can also be used for pull requests.
+
+* `Type: Refactor` - Issues that pertain to improving the existing code base.
+
+* `Type: Idea Bank` - Issues that pertain to proposing potential improvements to the code base.
+
+* `Type: Enhancement` - Issues marked as an agreed upon enhancement to the code base. Can also be used for pull requests.
+
+* `Status: Help wanted` - Issues where we need help from the greater open source community to solve.
+
+For a complete look at this repository's labels, see the
+[project labels page](https://github.com/charmed-hpc/slurm-charms/labels).
```

Comment:
Heh, got another one

Reply:
Rats, good catch. Need to up that Ctrl+F game :sob: